open_event/static/admin/js/map.js

```diff
@@ -1,13 +1,13 @@
 $( document ).ready(function() {
 
-    var long = $('#longitude').val();
-    var lati = $('#latitude').val();
-    var label_name = $('#name').val();
-    if (lati == '') {
-        lati = '51.08150695757747'
+    var long = $("#longitude").val();
+    var lati = $("#latitude").val();
+    var labelName = $("#name").val();
+    if (lati == "") {
+        lati = "51.08150695757747";
     }
-    if (long == ''){ long = '17.026073455812494'}
-    if (label_name == ''){ label_name = "Type name"}
+    if (long == ""){ long = "17.026073455812494"}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 4.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2418071464/issues/source?bid=3233864&fileBranchId=3237791#l9)